modules/ROOT/pages/index.adoc

```diff
@@ -5,16 +5,20 @@ endif::[]
 :imagesdir: ../assets/images/
 
 Object Store v2 lets CloudHub applications store data and states across batch processes
-and Mule components, from within a single application. Object Store v2 acts as the
+and Mule components, from within a single application.
+
+Object Store v2 acts as the
 backend for Mule 3 and Mule 4 CloudHub object store access, which enables applications
 to write directly to object store, use its REST API, and write and read using
 the Object Store connector. See the xref:osv2-faq.adoc[FAQ] for
 Object Store information.
 
-NOTE: Object Store v2 can only be used with CloudHub applications. CloudHub applications that use Mule 4 can enable use of Object Store v2 by setting the Runtime Manager or in-memory Object Store. On-premises Mule 4 applications do not use Object Store v2, but instead store data locally to a persistent data store. 
-
 image::osv2-architecture.png[width=300]
 
+Object Store v2 can only be used with CloudHub applications. CloudHub applications that use Mule 4 can enable the use of Object Store v2 by setting the Runtime Manager or in-memory Object Store.
```

Comment:
OSv2 can be used by both Mule 3 and Mule 4 apps.  To enable it users must check the box labeled `Use Object Store v2` in their application's settings in runtime manager.

Reply:
Thanks for the quick review. Updated.